### DISCUSSION TITLE
Unable to bind `cmd+.` to `esc:.` escape code

### DISCUSSION THREAD

**Discussion by denolfe** [ghostty-org/ghostty#3858] [Feature Requests, Ideas] [answered]
Hey Everyone,

I'm seeing some odd behavior when trying to bind `cmd+.` to escape code `esc+.` (pastes the last word of previous command).

I played with a bunch of different configurations enough so to think this is some sort of bug.

To recreate I ran through binding each one of the following one at a time:

```
# WORKS: Binding  cmd+. to an action
keybind = cmd+.=toggle_quick_terminal

# WORKS: Binding cmd+b to esc:.
keybind = cmd+b=esc:.

# DOES NOT WORK: Binding cmd+. to esc:.
keybind = cmd+.=esc:.
```

- Verified binding `cmd+.` to an action works
- Verified binding the escape code _to something other than `cmd+.` works_.
- When combining `cmd+.` with `esc:.`, no action occurs.

This is on version 1.0.0 running on macos Sonoma 14.6.1. Happy to provide any more detail if needed.

**Comment by 00-kat** [comment, marked as answer]
Dupe of #2881 I think.  Or at least, someone else also chimed in that `esc:` is also affected.

**Reply by denolfe** [reply]
Ah, this didn't show up in my search. Looks very similar, I agree. Thank you for finding that.

I'll add my details to that discussion 👍

**Reply by 00-kat** [reply]
Yep, `cmd` and `super` are aliases, so it makes sense that you didn't know that and hence didn't try searching for `super+.`.

**Comment by mitchellh** [comment]
#4146
